client/src/index.js

```diff
@@ -146,6 +148,17 @@ class MainView extends React.Component {
           taggedElementXPath={this.state.taggedElementXPath}
           unsetTaggedElement={this.unsetTaggedElement}
         />
+        <ToastContainer
+          position="bottom-center"
+          autoClose={5000}
+          hideProgressBar={false}
+          newestOnTop
+          closeOnClick
+          rtl={false}
```

Comment:
check that you are not declaring default values, i think this is false by default?